ada_feeding_actionselect/ada_feeding_actionselect/helpers.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+This module defines a number of helper functions that are reused throughout
+ada_feeding_actionselect.
+"""
+
+# Standard imports
+import os
+from typing import List
+import yaml
+
+# Third-party imports
+from ament_index_python.packages import get_package_share_directory
+
+# Local imports
+from ada_feeding_msgs.msg import AcquisitionSchema
+
+
+def get_action_library(
+    library_path: str, key: str = "actions"
+) -> List[AcquisitionSchema]:
+    """
+    Loads library for actions from a YAML file.
+
+    Parameters
+    ----------
+    library_path: location of yaml relative to package share
+    key: Must be an array of Acquisition Actions
+
+    Returns
+    -------
+    Array of AcquisitionSchema message objects
+    """
+
+    package_share = get_package_share_directory("ada_feeding_actionselect")
+    filename = os.path.join(package_share, library_path)
+
+    yaml_file = None
+    with open(filename, "r", encoding='utf-8') as file:
+        yaml_file = yaml.safe_load(file)
```

Comment:
I'd suggest putting this into a try-except and raising an informative error (e.g., `f"Failed to load acquisition library at {filename}. Are you passing in the right library parameter value? {exc}"`)